.eslintrc.cjs

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const { overrides } = require('@netlify/eslint-config-node')
+
+module.exports = {
+  extends: '@netlify/eslint-config-node',
+  parserOptions: {
+    sourceType: 'module',
+  },
+  rules: {
+    'import/extensions': [2, 'ignorePackages'],
+  },
```

Comment:
Once all our repositories have switched to ES modules, we can move this rule, and also `sourceType: 'module'` to `@netlify/eslint-config-node`.